### ISSUE TITLE
Mouth breathing

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#10363]
**Preferred term label:**
Mouth breathing

**Definition (free text, please give PubMed ID)**
Breathing through the mouth instead of the nose (PMID: 29799704).

**Comment**
In mouth breathing, about 25%−30% of the air passing through the mouth instead of the nose, that often occurs due to upper airway obstruction which reduces the nasal airflow and forces the air to enter completely or partially through oral cavity (PMID: 36159237). 

**Parent term (use hpo.jax.org/app)**
Abnormal pattern of respiration HP:0002793

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

**Comment by jagcsun** [comment]
However, a related term is already available - Nasal congestion HP:0001742.
